docs/hw/bus/pmc.rst

```diff
@@ -282,16 +282,26 @@ no effect and currently unknown purpose:
    Has the same bits as ENABLE, comes up as all-1 on boot, except for PDISPLAY
    bit which comes up as 0.
 
-.. reg:: 32 pmc-enable-unk0c ??? related to enable
+.. reg:: 32 pmc-elpg-enable ELPG enables
 
-   Has bits which correspond to PFIFO engines in ENABLE, ie.
+   Enable engine-level power gating (ELPG) controllers. Bits correspond to PFIFO
+   engines in ENABLE, ie.
 
    - 1: PPPP
+   - 2: PXBAR
```

Comment:
PXBAR, PFB and PHUB were each there in code headers as early as GK20A (don't have the hardware to verify use, but the code is pretty clear they are set).
The others I've personally verified as used from Pascal.